bundles/core/org.eclipse.smarthome.core.audio/src/main/java/org/eclipse/smarthome/core/audio/internal/AudioConsoleCommandExtension.java

```diff
@@ -83,6 +90,60 @@ public void execute(String[] args, Console console) {
         }
     }
 
+    private static final String SUBCMD_CREATE = "create";
+    private static final String SUBCMD_REMOVE = "remove";
+    private static final String SUBCMD_ADD = "add";
+    private static final String SUBCMD_CLEAR = "clear";
```

Comment:
If we keep that code into that file we should move that static finals to the other ones.
And IMHO we should also use that values in the buildCommandUsage, so if we change the SUBCMD constant, the documentation are changed to (similar to the SUBCMD_GROUPS etc.).
Perhaps the constants should be renamed from SUBCMD_xxx to SUBCMD_GROUP_xxx.